webapp/src/components/tutorial/TutorialContainer.tsx

```diff
@@ -85,7 +85,7 @@ export function TutorialContainer(props: TutorialContainerProps) {
         } else {
             setParentHeight();
         }
-    })
+    }, [props.hasBeenResized, document.body])
```

Comment:
Why do you need `props.hasBeenResized` here if you just return at the beginning of the `useEffect`?

Reply:
This is to make sure the function is run in the scenario where it changes from true to false.

Reply:
oh. That makes sense, thanks!

Reply:
(Admittedly, that's more of a theoretical scenario than anything else. It shouldn't really be changing in that direction...but I figured it didn't hurt to cover it, just in case something changes down the line)